Source/WarpX.cpp

```diff
@@ -1138,6 +1141,9 @@ WarpX::ReadParameters ()
         current_deposition_algo = static_cast<short>(GetAlgorithmInteger(pp_algo, "current_deposition"));
         charge_deposition_algo = static_cast<short>(GetAlgorithmInteger(pp_algo, "charge_deposition"));
         particle_pusher_algo = static_cast<short>(GetAlgorithmInteger(pp_algo, "particle_pusher"));
+        evolve_scheme = static_cast<short>(GetAlgorithmInteger(pp_algo, "evolve_scheme"));
+        utils::parser::queryWithParser(pp_algo, "max_picard_iterations", max_picard_iterations);
+        utils::parser::queryWithParser(pp_algo, "picard_iteration_tolerance", picard_iteration_tolerance);
```

Comment:
Should we raise an error if the `evolve_scheme` is not explicit, but the user chooses anything else than the electromagnetic Yee solver (e.g. electrostatic solver, or PSATD solver), since it seems that the code would in this case ignore the type of solver that was chosen.